bin/fuel-core/src/cli/run.rs

```diff
@@ -189,12 +189,13 @@ pub struct Command {
     #[arg(long = "native-executor-version", env)]
     pub native_executor_version: Option<StateTransitionBytecodeVersion>,
 
-    /// The starting gas price for the network
-    #[arg(long = "starting-exec-gas-price", default_value = "0", env)]
-    pub starting_exec_gas_price: u64,
+    /// The starting execution gas price for the network
+    #[arg(long = "starting-gas-price", default_value = "0", env)]
+    pub starting_gas_price: u64,
 
     /// The percentage change in gas price per block
-    #[arg(long = "gas-price-change-percent", default_value = "0", env)]
+    #[arg(long = "gas-price-change-percent", default_value = "2", env)]
```

Comment:
Sure, 10% just feels like such a huge number. This is already enough to triple the gas price within a minute (assuming 1s block times), and by 59k times within 10 minutes. I don't know how fast we want to be able to respond to changes, but it just feels like a lot.

But I'll set it to 10 to be consistent with our conception.

Reply:
Yeah. @xgreenx said 10% as a default. I'm of the opinion that that is high as well, since we have 1s blocks. But as long as we can give reasonable estimates, I don't think quickly changing prices is a bad thing.

Reply:
Done in https://github.com/FuelLabs/fuel-core/pull/2507/commits/02724a7ebf690cf9805a2cbb0e015ac0adf29bd2